packages/vite-plugin-cloudflare/playground/dev-vars/.gitignore

```
@@ -0,0 +1 @@
+!.dev.vars
```

Shall we do this in case others are added to the root `.gitignore` in the future.

```suggestion
!.dev.vars*
```

I'm wondering if we need to do this for `.env*` files that are used in playgrounds too now that we're in this repo?

---

> I'm wondering if we need to do this for .env* files that are used in playgrounds too now that we're in this repo?

We can, or we can just wait until the need arises, it is a single line addition anyways, we can do it at any point 👍